rwatch/ui/layer/scroll_layer.c

```diff
@@ -29,6 +29,8 @@ ScrollLayer *scroll_layer_create(GRect frame)
     return slayer;
 }
 
+PropertyAnimation *prop_anim;
```

Comment:
static

Reply:
Er, hmm.  This should presumably be a property of the ScrollLayer structure, right?